lib/element.js

```diff
@@ -161,6 +161,100 @@ ElementArrayFinder.prototype.all = function(locator) {
 };
 
 /**
+ * Get a slice of elements within the ElementArrayFinder.
+ * Negative indices are wrapped (i.e. -i means ith element from last)
+ * This does not actually retrieve the underlying element.
+ *
+ * Slice can be negative in which case we go backwards.
```

Comment:
Thank you for your review comments. One use case for the step is as follows : 
Let us say I have a plain text grid with 5 columns. 
Name  Age  DOB        Gender Status 
ABC    17     1/1/2011  M          Active
ABC1   10     1/6/2001  M          Inactive
ABC2   83     7/1/1931  M          Active
To extract out all elements for a particular column, I should be able to use slice with a start offset of the column number and step size of 5. So, to get all ages, I can call slice(1,undefined,5) assuming I can locate the parent element of the grid.

I hope this example is clear. I'm using protractor to automate the tests for my angular app and this is very handy to extract out column data from angular grids. There are other use cases as well.